frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerThread.java

```diff
@@ -178,7 +178,9 @@ public void run() {
         currentThread.set(thread);
         BaseModelRequest req = null;
         int status = HttpURLConnection.HTTP_INTERNAL_ERROR;
-
+        if (java.lang.System.getenv("SM_TELEMETRY_LOG_REV_2022_12") != null) {
+            loggerTelemetryMetrics.info("Telemetry enabled.");
+        }
```

Comment:
Why this log is needed at here?

Reply:
This logging message will help us to get endpoint name using TS for query total invoke account (in the case of no error log being generated.)